src/Anonymization/Anonymizer/Core/ConstantAnonymizer.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\Core;
+
+use MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\AbstractAnonymizer;
+use MakinaCorpus\DbToolsBundle\Attribute\AsAnonymizer;
+use MakinaCorpus\QueryBuilder\Query\Update;
+
+#[AsAnonymizer(
+    name: 'constant',
+    pack: 'core',
+    description: 'Set all value to \'xxxxxx\' or a custom value (with the \'value\' option).'
+)]
+class ConstantAnonymizer extends AbstractAnonymizer
```

Comment:
You probably should add a "type" option, which then will be propagated as a `new Value($value, $type)` (or `$expr->value($value, $type)`), doing so will allow the query-builder PHP to SQL value converter to do its work.

Reply:
done